docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,15 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger
```

Comment:
What does this mean? Sandbox?

Reply:
The _sandbox_ term is going to go away, so yes, it means the implementation of the ledger shipped with the SDK.

Reply:
Can this stay "Sandbox" until we officially change the name? (that should be in a release note and affects quite a lot of the docs, so let's do it in one go)

Reply:
Why do you think the Ledger heading is confusing? Especially considering that we use the "sandbox" terminology in the individual notes.

Reply:
Because that's not the term we use, and that means people have to work out what it means, rather than it being obvious because it's something we've used before, ie `sandbox`. Ledger could refer to a bunch of different things (eg are we talking about the ledger api?) in a heading like this.